src/main/java/seedu/address/ui/BinPersonCard.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import seedu.address.model.person.Person;
+
+/**
+ * An UI component that displays information of a {@code Person} in the bin.
+ */
+public class BinPersonCard extends PersonCard {
+
+    private static final String FXML = "BinPersonListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    @FXML
+    private Label dateDeleted;
+
+    @FXML
+    private Label expiryDate;
+
+    public BinPersonCard(Person person, int displayedIndex, String dateDeleted, String expiryDate) {
+        super(person, displayedIndex, FXML);
+        this.dateDeleted.setText(dateDeleted);
```

Comment:
Just curious, did you try 
```
super(person, displayedIndex)
super(FXML)
```
instead of overloading a constructor? Will the above work?

Reply:
yeah, it doesn't work sadly :(. From the error I've been getting (that the super class PersonCard does not have the constructor with only one parameter), it seems like `super` can only call the constructor of one class up. And also I don't think it's possible to call two super constructors at the same time, cos calling `super(person, displayedIndex)` already creates an object, so calling super another time is essentially creating two new objects?

Reply:
oo tks for explaining shud have taken 2030 more seriously.